benchmarks/bench_covertype.py

```diff
@@ -25,17 +25,18 @@
 
 The same task has been used in a number of papers including:
 
- * `"SVM Optimization: Inverse Dependence on Training Set Size"
-   <http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.139.2112>`_
```

Comment:
Now fixed all the citeseerx links and also put the doi's to the end of the references.

Reply:
Pls check rendering for feedback. ;)